cms/server/admin/static/training_program.js

```
@@ -0,0 +1,456 @@
+/* Contest Management System
+ * Copyright © 2024 IOI-ISR
+ *
+ * Training Program JavaScript Utilities
+ * Centralized JS for training program pages (histogram modal, etc.)
+ */
+
+"use strict";
+
+var CMS = CMS || {};
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)